src/backend/base/langflow/components/jigsawstack/text_to_sql.py

```
@@ -0,0 +1,88 @@
+            required=True,
+        ),
+        MessageTextInput(
+            name="sql_schema",
+            display_name="SQL Schema",
+            info="The database schema where the query will be run. Not required if file_store_key is specified.",
+            required=False,
+        ),
+        StrInput(
+            name="database",
+            display_name="Database",
+            info="The database type to generate SQL for. Supported values are postgresql, mysql, or sqlite. Specifying this parameter improves SQL generation accuracy by applying database-specific syntax and optimizations.",
```

Ruff (0.11.9)

Line too long (224 > 120)

(E501)